'

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.

```

Comment:
Is this file name intentional?

Reply:
No, I don't know how this got created. I don't know if it's a bug - I've encountered instances where similar files were generated randomly while using Visual Studio.

Reply:
Removed it.